docs/release-notes.md

```diff
@@ -1,6 +1,18 @@
 # GA Releases
 
-This page summarizes changes in each major update in Timeplus, including new features and important bug fixes.
+This page summarizes changes for each major update in Proton and Timeplus Cloud, including new features and important bug fixes.
+
+## Oct 16, 2023
+
+*Proton:*
+  * New data types now supported: ipv4 and ipv6, as well as related (functions)[functions_for_url].
+  * [Python Driver](https://github.com/timeplus-io/proton-python-driver) 0.2.9 now supports Python 3.11 and 3.12.
```

Comment:
Qijun just added a new version 0.2.10 to refine README. So let's replace 0.2.9 with 0.2.10